client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -2,22 +2,21 @@ import Button from '@material-ui/core/Button';
 import Box from '@material-ui/core/Box';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
-import { Typography } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
 
 interface Props {
   linkTo: string;
-  asideText: string;
   btnText: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
     <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
-      <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
+      <Typography color="secondary" className={classes.tattooArt}>TATTOO ART</Typography>
```

Comment:
Some screen readers pickup capital case text as acronyms, consider using the tattooArt class to transform the text to uppercase :)

Reply:
This is fixed.